src/courses.json

```diff
@@ -26,15 +26,15 @@
   "Informationsmanagement": "IM",
   "Bachelor-Seminar: Digitale Hochschule: Aktuelle Trends und Herausforderungen": "Digitale Hochschule",
   "Betriebssysteme und Systemsoftware": "BS",
-  "Einführung in die Informatik ": "INFO",
+  "Einführung in die Informatik ": "EIDI",
   "Praktikum: Grundlagen der Programmierung": "PGdP",
   "Einführung in die Rechnerarchitektur": "ERA",
   "Grundlagenpraktikum: Rechnerarchitektur": "GRA",
   "Einführung in die Softwaretechnik": "EIST",
   "Grundlagen: Algorithmen und Datenstrukturen": "GAD",
   "Effiziente Algorithmen und Datenstrukturen": "EAD",
-  "Rechnernetze und Verteilte Systeme": "RNVS",
-  "Einführung in die Theoretische Informatik": "THEO",
+  "Rechnernetze und Verteilte Systeme": "GRNVS",
```

Comment:
![image](https://user-images.githubusercontent.com/26258709/165635881-68637501-e86e-4227-a1b9-ba3e16444e87.png)
There are other courses that exist, with this Name being a subset.

since we are doing literal string replacement, `Grundlagen: Rechnernetze und Verteilte Systeme` would probably (have not tested) become `G: GRNVS`

I have not looked at enough php in my live to know, if a duplicate, ie
```suggestion
  "Grundlagen: Rechnernetze und Verteilte Systeme": "GRNVS",
  "Rechnernetze und Verteilte Systeme": "RNVS",
```
would work

Reply:
Good catch. This is possible, indeed. I just tested it. PHP iterates through the array.